react/src/Components/RestaurantIndexTile.js

```diff
@@ -2,8 +2,19 @@ import React from 'react'
 
 const RestaurantIndexTile = props => {
   return(
-    <div>
-      <h3><a href={`/restaurants/${props.id}`}>{props.name}</a></h3>
+    <div className={'row small-8 columns small-offset-2 index-container end'}>
```

Comment:
For all these, if you are not interpreting anything in the `className` (i.e., it's just a string), you can just keep it as a string without the curly braces.

So for example, this line could be:

`<div className="row small-8 columns small-offset-2 index-container end">`